server/config.js

```diff
@@ -0,0 +1,30 @@
+// Copy this file to config.js and change values accordingly
```

Comment:
Even though you added config.js back to the gitignore, you need to remove this file from the project because you already committed it. https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore